### DISCUSSION TITLE
Two things I have no idea how to do it

### DISCUSSION THREAD

**Discussion by tollfree** [rxhanson/Multitouch-Community#169] [Q&A]
HI folks,
Here is a couple of options I'm trying to add to my life and can't yet realize:

1. A gesture to open a new Safari tab from anywhere and switch to it;
2. Enter Picture in Picture YouTube/any video mode.

The first one can be realized in BTT, and maybe there is a way to do it here, but the second doesn’t even have its own hotkey.

Thanks in advance for any advice

**Comment by rxhanson** [comment]
> 1. A gesture to open a new Safari tab from anywhere and switch to it;

Your best option would be to script this in AppleScript and execute a bash command to call the AppleScript.

Alternatively, you could set up something like rest four, remove all but one to open safari, then rest four, remove all but one and move it down to open a new tab. Then you can perform those gestures together to open safari then open a new tab.

> 2. Enter Picture in Picture YouTube/any video mode.

Without a hotkey for this, I can't think of a good way to do it off the top of my head. I'll give it some more thought though.
